verify-shellcheck.sh

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env bash
```

Comment:
Can you add a README describing what are we validating with this script?

Reply:
Okay.